docs/notifications.md

```diff
@@ -27,11 +27,13 @@ Currently supported events:
 Filters use conjunctional logic.
 
 ## Ordering and persistence guarantees
- * new block is only announced after its processing is complete and the chain
-   is updated to the new height
+ * new block and header of this block are only announced after block's processing
+   is complete and the chain is updated to the new height
  * no disk-level persistence guarantees are given
+ * header of newly added block is announced after block processing, but before 
+   announcing the block header and the block itself
```

Comment:
Should be:
* header of newly added block is announced after block processing, but before 
   announcing the block itself